includes/commerce_pos.transaction.inc

```diff
@@ -936,3 +937,80 @@ function _commerce_pos_override_confirm_validate($form, &$form_state) {
   // Add the product to the overridden items list.
   $form_state['out_of_stock_overridden_products'][$form_state['triggering_element']['#product_id']] = $form_state['triggering_element']['#product_id'];
 }
+
+/**
+ * Returns a modal form displaying each item in an order along with a checkbox
+ * for the user to add that item to the return transaction.
+ */
+function commerce_pos_return_select_item_popup(&$form, &$form_state, $order) {
+
+  // Load our generic modal form.
+  $form['commerce_pos_return_select_item_modal'] = commerce_pos_form_modal(array(
```

Comment:
Yeah, I didn't give too much thought into it. Travis would be able to answer this quickly as he did the restructuring for the generic modal.